sdks/python/src/opik/api_objects/opik_client.py

```diff
@@ -1121,6 +1130,34 @@ def get_optimization_by_id(self, id: str) -> optimization.Optimization:
         _ = self._rest_client.optimizations.get_optimization_by_id(id)
         return optimization.Optimization(id=id, rest_client=self._rest_client)
 
+    def optimization_id(
```

Comment:
optimization_id is quite confusing and misleading, let's go with optimization_context